content/resurrector.md

```diff
@@ -156,12 +145,79 @@ rules:
+- `enabled` - a boolean which enables (`true`) or disables (`false`) resurrection.
+- `exclude` *(Optional)* - a resurrection rule which will result in the resurrection configuration being overridden wherever the specified constraints do not match.
+- `include` *(Optional)* - a resurrection rule which will result in the resurrection configuration being overridden wherever the specified constraints match.
+- `deployments` *(Optional)* - a list of deployments which can be used as a filter for the include/exclude directives.
+- `instance_groups` *(Optional)* - a list of instance groups which can be used as a filter for the include/exclude directives.
+
+
+### General
+
+* When specifying both properties, `instance_groups` and `deployments`, the rule is only applied for the instance groups of the specified deployments.
+* When only `instance_groups` is specified, the rule will be applied to all matching instance groups across **all** deployments.
+* Multiple rules are evaluated by the `&` operator. This means, if one rule with `enabled: false` exists, all deployments and instance groups which are included in this rule have resurrection disabled.
+
+
+### Examples
+
+By default, resurrection is turned on and the following examples demonstrate how this default can be overwritten.
+
+1. Disable resurrection for a deployment `dep1` by creating a `resurrection.yml`:
+
+	```yaml
+	rules:
+	- enabled: false
+	  include:
+	    deployments:
+	    - dep1
+	```
+	Running `bosh update-config --type resurrection --name default resurrection.yml` disables resurrection for all deployments in the include block, i.e. `dep1`. For all other deployments, resurrection is still enabled.
+
+2. Disable resurrection for an instance group `instance-group-1` of a deployment `dep1` by creating a `resurrection.yml`:
+
+	```yaml
+	rules:
+	- enabled: false
+	  include:
+	  	 deployments:
+	  	 - dep1
+	    instance_groups:
+	    - instance-group-1
+	```
+	
+	Running `bosh update-config --type resurrection --name default resurrection.yml` disables resurrection for all instance groups of all the specified deployments, i.e. `instance-group-1` of deployment `dep1`. For all other instance groups and deployments, resurrection is still enabled.
+
+3. Disable resurrection for all deployments except for deployment `dep1` by creating a `resurrection.yml`:
+	
+	```yaml
+	rules:
+	- enabled: false
+	  exclude:
+	    deployments:
+	    - dep1
+	```
+	Running `bosh update-config --type resurrection --name default resurrection.yml` disables resurrection for all deployments except deployment `dep-1`.
+
+
+### Disabling resurrection globally
+
+Resurrection can be disabled on all deployments with this config:
+
+```yaml
+rules:
+- enabled: false
+```
+
+One liner for turning resurrection off globally:
+
+```sh
+bosh update-config --type resurrection --name default <(echo "{\"rules\": [{\"enabled\": false}]}")
```

Comment:
Do we want to use the name `default` here? If users have already some configuration uploaded, this will override it, correct? Suggestion: use some other clearly defined name, such as `bosh.global.disable` or similar.

Reply:
Sounds good.